### PR TITLE
Re-org cruise tuning info a little

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: plane/source/docs/tuning-cruise.rst [hunk 1]
```diff
@@ -10,32 +10,54 @@ Overview of Constant Altitude, Level Flight Operation
...
 
 Adjusting FBWB or CRUISE Mode Airspeed
 ======================================
 
 The autopilot's goal in automatic throttle modes is to obtain the correct combination of elevator and throttle to maintain constant altitude flight. How the autopilot does this is detailed in :ref:`tecs-total-energy-control-system-for-speed-height-tuning-guide`.
 
-When using an airspeed sensor in FBWB or CRUISE, the target airspeed can be directly controlled with the throttle stick position. Mid throttle will set the speed as halfway between :ref:`ARSPD_FBW_MAX<ARSPD_FBW_MAX>` (high stick) and :ref:`ARSPD_FBW_MIN<ARSPD_FBW_MIN>` (low stick). Using :ref:`FLIGHT_OPTIONS<FLIGHT_OPTIONS>` bit 3 you can force the target airspeed in these modes to always be :ref:`TRIM_ARSPD_CM<TRIM_ARSPD_CM>` ignoring throttle position. Alternatively, Bit 1 of :ref:`FLIGHT_OPTIONS<FLIGHT_OPTIONS>` can be set to make the center throttle stick position be :ref:`TRIM_ARSPD_CM<TRIM_ARSPD_CM>` instead of midway between :ref:`ARSPD_FBW_MAX<ARSPD_FBW_MAX>` (high stick) and :ref:`ARSPD_FBW_MIN<ARSPD_FBW_MIN>` (low stick).
+When Airspeed Sensor Equipped
```
Comment (TunaLobster) [inline plane/source/docs/tuning-cruise.rst:37]:
```suggestion
When an Airspeed Sensor is Enabled
```

File: plane/source/docs/tuning-cruise.rst [hunk 2]
```diff
@@ -10,32 +10,54 @@ Overview of Constant Altitude, Level Flight Operation
...
+
+
+Without an Airspeed Sensor
+--------------------------
+
+Without an airspeed sensor, both the pitch trim and the :ref:`TRIM_THROTTLE<TRIM_THROTTLE>` parameter would need to be changed appropriately for the desired mid-stick cruise speed. 
+
+.. tip:: Often small light planes or gliders need 2 or 3 degrees of pitch trim to fly at their optimum cruising speed/throttle rather than at the fuselage/autopilot level pitch. This can be done at setup by:
+
+- **(Preferred)** Add the desired degrees nose up(usually) or down to :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>`.
```
Comment (TunaLobster) [inline plane/source/docs/tuning-cruise.rst:54]: These bullet points are not exclusive to small aircraft. They are important for all aircraft.

File: plane/source/docs/tuning-cruise.rst [hunk 3]
```diff
@@ -10,32 +10,54 @@ Overview of Constant Altitude, Level Flight Operation
...
+
+.. tip:: You can examine ATT.Pitch in the logs when at cruise speed in FBWB or CRUISE to determine the average pitch trim required in these modes. Appropriately adjusting :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>` to lower this to zero when flying level in these modes.
+
+.. note:: when using :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>` to adjust trim, it will be reflected in the ATT.Pitch log message, and also in the OSD and GCS horizon displays (ie level horizon pitch = :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>` + the calibrated level pitch) so that the display will be level when flying "in trim" even though the plane's pitch is different than the autopilots calibrated pitch. If it is desired to actually reflect the autopilots pitch, use :ref:`FLIGHT_OPTIONS<FLIGHT_OPTIONS>` bits 8 and/or 9  to show this in OSD and/or GCS artificial horizon.
```
Comment (TunaLobster) [inline plane/source/docs/tuning-cruise.rst:60]: Did those options make it into a stable release?

File: plane/source/docs/tuning-cruise.rst [hunk 4]
```diff
@@ -10,32 +10,54 @@ Overview of Constant Altitude, Level Flight Operation
...
+The AOA is usually several degrees but varies depending on cruise speed/throttle. This is the "trim level" pitch. This is explained in the diagrams above and is known as the "trim level" condition. While many planes will have some Angle of Incidence (i.e. the cord of the wing is at a positive angle to the fuselage cord) built-in, some do not, and some need a slightly higher AOA to fly at desired cruise speeds.
 
-.. image:: ../../../images/AOA.jpg
+If the level step of calibration is done with the plane's fuselage line level, initial flights will be safe, but the aircraft may not hold altitude at the desired flying speed in non-altitude controlled modes (i.e. fly too fast and/or require too much throttle to hold altitude) and/or the cruising speed in non airspeed sensor equipped planes may be faster or slower than desired.
```
Comment (TunaLobster) [inline plane/source/docs/tuning-cruise.rst:30]: Move the closing parenthesis to the end.

File: plane/source/docs/tuning-cruise.rst (hunk 1 above)
Comment (Hwurzburg) [reply plane/source/docs/tuning-cruise.rst:37]: better..

File: plane/source/docs/tuning-cruise.rst (hunk 3 above)
Comment (Hwurzburg) [reply plane/source/docs/tuning-cruise.rst:60]: no,and already in Upcoming...will remove for this

File: plane/source/docs/tuning-cruise.rst (hunk 4 above)
Comment (Hwurzburg) [reply plane/source/docs/tuning-cruise.rst:30]: yep...what I meant

Comment (Hwurzburg) [reply plane/source/docs/tuning-cruise.rst:30]: reworded

File: plane/source/docs/tuning-cruise.rst (hunk 2 above)
Comment (Hwurzburg) [reply plane/source/docs/tuning-cruise.rst:54]: rewording a bit to allow it to apply to all

**Pull request merged**